path_checkouts_test.go

```diff
@@ -0,0 +1,30 @@
+package openldap
```

Comment:
This is copied from [vault-plugin-secrets-ad](https://github.com/hashicorp/vault-plugin-secrets-ad).